python/chronos/script/benchmark-chronos.py

```diff
@@ -171,7 +171,24 @@ def latency():
         records['openvino_percentile_latency'] = np.percentile(latency_vino, latency_percentile)
 
 
+def evaluation():
+    """
+    evaluate stage will record model accuracy.
+    """
+
+    forecaster.fit(train_loader, validation_data=val_loader, epochs=args.training_epochs, validation_mode="best_epoch")
```

Comment:
recently I found "best_epoch" might cause some issue like "val/loss" not found. Please pay attention if same issue happens to you

Reply:
OK